### PR TITLE
Add signature of gettimeofday js library function

### PR DESCRIPTION
**Pull request by ravisumit33**
Fixes: #14959

### CODE REVIEW

**Comment by ravisumit33** [comment]
@sbc100 Please review

File: src/library.js [hunk 1]
```diff
@@ -1452,6 +1452,7 @@ LibraryManager.library = {
     if (clk_id) {{{ makeSetValue('clk_id', 0, 2/*CLOCK_PROCESS_CPUTIME_ID*/, 'i32') }}};
     return 0;
   },
+  gettimeofday__sig: 'ii',
```
Comment (sbc100) [inline src/library.js:1455]: I think the sig of this function is `iii` (return an int and takes 2 pointers).

Comment (ravisumit33) [reply src/library.js:1455]: May be I am wrong, but I can see only one argument `ptr` in the function gettimeofday ([here](https://github.com/emscripten-core/emscripten/blob/main/src/library.js#L1456))

Comment (sbc100) [reply src/library.js:1455]: It take a second argument which is the timezone (which this version currently completely ignores I guess): https://man7.org/linux/man-pages/man2/settimeofday.2.html.

Comment (ravisumit33) [reply src/library.js]: Updated

**Review by sbc100 — APPROVED** [review]

**Comment by ravisumit33** [comment]
@sbc100 Failing tests shows context deadline exceeded. Any idea what can cause that ?

**Comment by sbc100** [comment]
That's test flakiness, I restarted the test run.

**Pull request merged**